app/src/processing/app/Base.java

```diff
@@ -1264,7 +1264,7 @@ public void actionPerformed(ActionEvent e) {
     return found;
   }
 
-
+  
```

Comment:
Please don't add extra whitespace. If anything, have your editor trim it from the end of lines.